src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -290,6 +290,29 @@ def CheckTestCommissionerCallbacks(self):
     def CheckTestCommissionerPaseConnection(self, nodeid):
         return self._dmLib.pychip_TestPaseConnection(nodeid)
 
+    def CommissionWithCode(self, setupPayload: str, nodeid: int):
```

Comment:
maybe we can start from here to remove ChipDeviceCtrl.py and start to create the Commision.py?

Reply:
Based on the use of devCtrl class, I would suggest a separate refactor to remove the legacy ZCL code first.